src/api.raml

```diff
@@ -260,21 +260,7 @@ traits:
                 description: |
                     List of annual reports, sorted by year.
                 body:
-                    application/vnd.elife.annual-report-list+json;version=1:
-                        schema: !include ../dist/model/annual-report-list.v1.json
-                        examples:
-                            populated:
-                                displayName: Populated list
-                                description: |
-                                    List is split over two pages.
-                                value: !include samples/annual-report-list/v1/populated.json
-                            empty:
-                                displayName: Empty list
-                                description: |
-                                    Demonstration of the list being empty (the first page will always return a valid result).
-                                value: !include samples/annual-report-list/v1/empty.json
                     application/vnd.elife.annual-report-list+json;version=2:
-                        (experimental):
```

Comment:
If anyone else was using this we should mark the other as deprecated prior to removing. But they're not.